src/components/plan/PlanForm.jsx

```diff
@@ -88,117 +93,93 @@ const PlanForm = () => {
     };
 
     return (
-        <section className="
-                text-gray-600
-                body-font
-                relative
-        ">
-            <div className="
-                container
-                px-5
-                py-24
-                mx-auto
-            ">
-                <div className="
-                    flex
-                    flex-col
-                    text-center
-                    w-full
-                    mb-12
-                ">
-                    <h1 className="
-                        sm:text-3xl
-                        text-2xl
-                        font-medium
-                        title-font
-                        mb-4
-                        text-gray-900
-                    ">
-                        Create a new plan
-                    </h1>
-                    <p className="
-                        lg:w-2/3
-                        mx-auto
-                        leading-relaxed
-                        text-base
-                    ">
-                        Fill out the form below to create a new plan.
-                    </p>
+        <section className="text-gray-600 body-font ">
+            {loading && (
```

Comment:
I think you don't have to remove `relative` because can change its behaviour. The problem is that you're putting the loader inside this `section` when it doesn't have to.

Reply:
Solved